pypeit/core/wavecal/waveio.py

```diff
@@ -109,20 +111,47 @@ def load_template(arxiv_file, det):
     # Return
     return tbl['wave'].data[idx], tbl['flux'].data[idx], tbl.meta['BINSPEC']
 
+
 def load_reid_arxiv(arxiv_file):
+    """
+    Load a REID arxiv file
+    Now there are 2 possible formats.  We need to consolidate
+
+    Args:
+        arxiv_file (str):
+
+    Returns:
+        dict, dict-like:
+
+    """
     # ToDO put in some code to allow user specified files rather than everything in the main directory
     calibfile = os.path.join(reid_arxiv_path, arxiv_file)
-    wv_calib_arxiv = load_wavelength_calibration(calibfile)
-    par = wv_calib_arxiv['par'].copy()
-    # Pop out par and steps if they were inserted in this calibration dictionary
-    try:
-        wv_calib_arxiv.pop('steps')
-    except KeyError:
-        pass
-    try:
-        wv_calib_arxiv.pop('par')
-    except KeyError:
-        pass
+    # This is a hack as it will fail if we change the data model yet again for wavelength solutions
```

Comment:
I thought the idea was that you were going to remove the json archives, and do everything with these new fits files?

Reply:
easy to remove them if we want.